README.md

```diff
@@ -160,13 +163,13 @@ The following commands are the minimal set of commands to get setup for running
 4. Run the pipeline
 
     ```bash
-    mti_nma all run --path_blender /allen/aics/modeling/jacksonb/applications/blender-2.82-linux64/blender --nsamples {int} --distributed
+    mti_nma all run --path_blender /allen/aics/modeling/jacksonb/applications/blender-2.82-linux64/blender --nsamples {int} --distributed True
```

Comment:
You actually don't need the `True` at the end. Simply by passing `--distributed` it should assume True